javabot/PokerUtilities.java

```diff
@@ -0,0 +1,51 @@
+enum HandType {
+  ERROR (0),
+  HIGH_CARD (1),
+  PAIR (2),
+  TWO_PAIR (3),
+  THREE_OF_A_KIND (4),
+  STRAIGHT (5),
+  FLUSH (6),
+  FULL_HOUSE (7),
+  FOUR_OF_A_KIND (8),
+  STRAIGHT_FLUSH (9 );
```

Comment:
```suggestion
  STRAIGHT_FLUSH (9);
```
Just call me the human linter